src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -102,7 +103,7 @@ internal static void Initialize()
         private EventPipeController()
         {
             // Set the config file path.
-            m_configFilePath = Path.Combine(AppContext.BaseDirectory, BuildConfigFileName());
+            m_configFilePath = Path.Combine(AppContext.BaseDirectory!, BuildConfigFileName());
```

Comment:
Might be worth including a comment here, e.g.
`// BaseDirectory could be null, in which case this could throw, but it will be caught and ignored: https://github.com/dotnet/coreclr/issues/24053`